plugins/kubeval.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+￼kind: Plugin
+￼metadata:
+￼  name: validate-manifests
+￼spec:
+￼  version: "0.14.0"
+￼  shortDescription: Validate your Kubernetes configuration files, supports multiple Kubernetes versions
```

Comment:
> supports multiple Kubernetes versions

part is redundant for a "short" description, it's better to actually keep it short.

Reply:
you can also use the `description` field if necessary.